script/c88774734.lua

```diff
@@ -61,14 +61,15 @@ function s.lvfilter(c)
 end
 	--Activation legality
 function s.lvtg(e,tp,eg,ep,ev,re,r,rp,chk)
-	if chk==0 then return Duel.IsExistingMatchingCard(s.lvfilter,tp,LOCATION_MZONE,0,1,e:GetHandler()) end
+	if chk==0 then return Duel.IsExistingMatchingCard(s.lvfilter,tp,LOCATION_MZONE,0,1,nil) end
```

Comment:
Spiral Dragon Bulge: Can activate as the only monster you control if its level is not 8